apps/logo-quiz/src/main.tsx

```diff
@@ -1,6 +1,21 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
+import { Route, BrowserRouter as Router } from 'react-router-dom';
 
 import { App } from "./app/app";
+import { LevelList } from './app/views/LevelList/LeveLlist';
+import { LogoVerify } from './app/views/LogoVerify/LogoVerify';
+import { LogoList } from './app/views/LogoList/LogoList';
 
-ReactDOM.render(<App />, document.querySelector("logo-quiz-root"));
+const routing = (
+  <Router>
+    <div>
+      <Route exact path="/" component={App}/>
+      <Route path="/logos/:id" component={LogoVerify}/>
+      <Route path="/levels/:id" component={LogoList}/>
+      <Route exact path="/levels" component={LevelList}/>
+    </div>
+  </Router>
+);
+
+ReactDOM.render(routing, document.querySelector("logo-quiz-root"));
```

Comment:
are we going to use single quotes or double?

```suggestion
ReactDOM.render(routing, document.querySelector('logo-quiz-root'));
```